src/uportQRDisplay.js

```diff
@@ -1,26 +0,0 @@
-import { Uport } from './uport'
-import { QRUtil } from './util/qrdisplay'
-
-
-class UportQRDisplay extends Uport {
-  constructor(dappName, opts = {}) {
-    super(dappName, opts)
-    this.openQR = QRUtil.openQr
-    this.closeQR = QRUtil.closeQr
-  }
-
-  request({uri, topic}) {
-    // TODO need self reference
-    console.log(this)
-    this.openQR(uri)
-    return new Promise((resolve, reject) => {
-      this.msgServer.waitForResult(topic, (err, res) => {
-        this.closeQR()
-        if (err) reject(err)
-        resolve(res)
-      })
-    })
-  }
-}
-
-export { UportQRDisplay }
```

Comment:
Yes subclassing can be a good solution, for somethings. But it didn't seem right here. I think the better fix is to do something with the build process, disabling it. Still working it out, but I didn't want to go out with it as is.